app/controllers/shipit/api/stacks_controller.rb

```diff
@@ -78,6 +84,37 @@ def stack
         @stack ||= stacks.from_param!(params[:id])
       end
 
+      def update_lock
```

Comment:
There is already an API to lock/unlock stacks https://github.com/Shopify/shipit-engine/blob/master/app/controllers/shipit/api/locks_controller.rb

Reply:
Got it, thank you!